filebeat/datadog_checks/filebeat/filebeat.py

```diff
@@ -221,11 +221,14 @@ class FilebeatCheck(AgentCheck):
     def __init__(self, *args, **kwargs):
         AgentCheck.__init__(self, *args, **kwargs)
         self.instance_cache = {}
+        self.tags = []
 
-        # preserve backwards compatible default timeouts
-        if self.instance and self.instance.get('timeout') is None:
-            if self.init_config.get('timeout') is None:
-                self.instance['timeout'] = 2
+        if self.instance:
+            self.tags = self.instance.get('tags', [])
+            # preserve backwards compatible default timeouts
+            if self.instance.get('timeout') is None:
+                if self.init_config.get('timeout') is None:
+                    self.instance['timeout'] = 2
```

Comment:
```suggestion
        self.tags = self.instance.get('tags', [])
        # preserve backwards compatible default timeouts
        if self.instance.get('timeout') is None:
            if self.init_config.get('timeout') is None:
                self.instance['timeout'] = 2
```
We don't need to check that actually

Reply:
@coignetp i did that initially, but that causes the test for invalid configs to fail. Makes sense because if an empty config is passed, `self.instance` does not exist.

Reply:
It does make sense, but it should not happen based on how integrations instances are created. However, we can keep it as it only happens in the constructor, thank you for your investigation